modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -146,6 +134,32 @@ code) and the Market app is enabled, but there is no available internet
 connection, then these apps will need to be manually updated once the
 upgrade is finished.
 
+[[copy-old-updated-apps]]
+== Copy Old Updated Apps
+
+If your setup has a read-only `apps/` folder and is using a separate
+writable folder for updated apps like `apps-external/` or another location,
+please skip this section as well.
+
+If you are installing the standard owncloud community bundle, usually
```

Comment:
s/owncloud/ownCloud/g

Reply:
Plus, it's likely good to draw attention to this and the paragraph above by using admonitions.

Reply:
@settermjd not sure how to do the admonition part. Add an addition NOTE: block at the beginning of the section ? What to write to draw attention ?

Reply:
[Here's the docs for admonitions](https://asciidoctor.org/docs/user-manual/#admonition).

Reply:
or here in a condensed version :smile:
https://github.com/owncloud/docs/blob/master/docs/best-practices.md#admonition